### PR TITLE
Add new missions that trigger surge pricing/limiter at high volume traffic

### PR DESCRIPTION
**Pull request by dmkozh**
This is based on the capability to generate load and ignore low fee tx errors (https://github.com/stellar/stellar-core/pull/3492)

I'm adding the following scenarios:
- Regular load generation, but with limiting/surge pricing triggering
- Protocol upgrade under load/limiting - this would be relevant for e.g. p20 upgrade to use generalized tx sets
- Running the mixed image version network at 51% quorum and with load - this could have caught network forking of 19.3.0 release.

### CODE REVIEW

File: src/FSLibrary/StellarKubeSpecs.fs
Comment (marta-lokhova) [inline src/FSLibrary/StellarKubeSpecs.fs:695]: I think this commit can be dropped now that we merged https://github.com/stellar/supercluster/pull/62. Github checks should also be fixed now (we had a super old image hardcoded there, so it was failing to parse the new flag)

File: src/FSLibrary/StellarMission.fs
Comment (marta-lokhova) [inline src/FSLibrary/StellarMission.fs:51]: Did you also want to add these new missions to the CI?

File: src/FSLibrary/MissionMixedImageLoadGeneration.fs
Comment (marta-lokhova) [inline src/FSLibrary/MissionMixedImageLoadGeneration.fs:30]: minor: would a setup of 4 nodes with default threshold achieve the same result? Then we wouldn't need the additional threshold functionality.

File: src/App/Program.fs
Comment (marta-lokhova) [reply src/App/Program.fs:150]: oh, is it to test different combinations of old vs new images? I would argue that it's better to just hardcode the desired parameters in the mission so we don't rely on the user configuring the mission correctly (as with this parameter changing, we end up testing very different things).

**Review by marta-lokhova — COMMENTED** [review]
looks good! just had a few questions/suggestions

File: src/FSLibrary/MissionMixedImageLoadGeneration.fs
Comment (dmkozh) [reply src/FSLibrary/MissionMixedImageLoadGeneration.fs:18]: I'm new to F#, but it seemed like I can't skip field when initializing a type. Also I don't really see the code that would populate keys in case if I provided an empty array. I would appreciate some example to look at in case if I'm missing something.

File: src/FSLibrary/MissionMixedImageLoadGeneration.fs
Comment (dmkozh) [reply src/FSLibrary/MissionMixedImageLoadGeneration.fs:30]: I guess the question is whether probability of 3 nodes simultaneously arriving at the same value (e.g. `txSetHash` for our case) is significantly different from probability of 2 nodes doing the same. I don't really know the answer, do you have any idea? Also, my other motivation was to have the same setup as testnet.

File: src/FSLibrary/MissionMixedImageLoadGeneration.fs
Comment (marta-lokhova) [reply src/FSLibrary/MissionMixedImageLoadGeneration.fs:18]: Sorry I wasn't super clear - you can use `MakeLiveCoreSet` which populates keys based on node count. For example, like we do here: https://github.com/stellar/supercluster/blob/65c2e47104a76417f141b92f6aaf706cd1d1455a/src/FSLibrary/MissionComplexTopology.fs#L17

File: src/App/Program.fs
Comment (marta-lokhova) [reply src/App/Program.fs:150]: I think two missions would be better tbh, so we can hardcode the exact scenarios we want to test. And I think we can refactor some common functionality to avoid duplication.

File: src/FSLibrary/MissionMixedImageLoadGeneration.fs
Comment (marta-lokhova) [reply src/FSLibrary/MissionMixedImageLoadGeneration.fs:30]: Yeah, it's true that it's probably better to replicate the testnet setup (for possible future issues as well). I think we can just keep your change as-is then: my original comment was around whether we needed the new code, but it's not much code to begin with, so not a big deal.

File: src/FSLibrary/StellarMission.fs
Comment (marta-lokhova) [reply src/FSLibrary/StellarMission.fs:51]: yeah, wanted to make sure you're aware that another change is needed elsewhere

File: src/FSLibrary/MissionMixedImageLoadGeneration.fs
Comment (dmkozh) [reply src/FSLibrary/MissionMixedImageLoadGeneration.fs]: Thanks for the pointer, updated the code to use this.

File: src/App/Program.fs
Comment (dmkozh) [reply src/App/Program.fs:150]: Sounds good, it's actually more compact if we keep both setups in the same file.

File: src/FSLibrary/StellarMission.fs
Comment (dmkozh) [reply src/FSLibrary/StellarMission.fs:51]: Sounds good, I'll keep this in mind while it's blocked by the core PR release.

**Review by jeffreysmooths — APPROVED** [review]

**Review by marta-lokhova — APPROVED** [review]
thank you for the updates, this looks great to me! We'll merge this once we land the prerequisite changes in core.

**Comment by dmkozh** [comment]
> thank you for the updates, this looks great to me! We'll merge this once we land the prerequisite changes in core.

Core changes have been finally merged (https://github.com/stellar/stellar-core/pull/3492), so this should be ready to be merged, PTAL

File: src/FSLibrary/StellarCoreHTTP.fs
Comment (marta-lokhova) [inline src/FSLibrary/StellarCoreHTTP.fs:69]: need to update this to be consistent with the new format (true/false instead of 1/0)

Comment (dmkozh) [reply src/FSLibrary/StellarCoreHTTP.fs]: Thanks for the reminder, fixed

File: src/FSLibrary/StellarMission.fs
Comment (marta-lokhova) [reply src/FSLibrary/StellarMission.fs:51]: heads up, we can update the CI now

File: src/FSLibrary/StellarCoreHTTP.fs
Comment (jeffreysmooths) [reply src/FSLibrary/StellarCoreHTTP.fs]: Good

**Pull request merged**
